rules/align.smk

```diff
@@ -42,19 +42,25 @@ def input_fq_args(fastqs):
     else:
         return f"-1 {fqs[0]} -2 {fqs[1]}"
 
+def check_sra():
+    if 'sra' in config and config["sra"] is not None:
```

Comment:
Could just throw in an `and` between these two boolean statements and return that :)